content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -233,24 +221,19 @@ portal.
    :align: center
    :alt: Customer preview of a quotation template in Odoo Sales.
 
-.. tip::
-   Quotation template design uses the same methodology and functionality with design building blocks
-   as a typical web page design with Odoo *Website*. Be sure to check out the
-   :doc:`../../../websites/website` documentation to learn more.
-
-When all blocks and customizations are complete, click the :guilabel:`Save` button to put those
-configurations into place.
+When all blocks and customizations are complete, click the :guilabel:`Save` button to save the
+configuration.
 
-There is also a blue banner at the top of the quotation template design with a link to quickly
-return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
-back-end of the *Sales* application.
+The blue banner located at the top of the quotation template preview can be used to quickly return
+:icon:`fa-arrow-right` :guilabel:`(Back to edit mode)`. When clicked, Odoo returns to the quotation
```

Comment:
Again no need for parenthesis here since the button has text and an icon
```suggestion
:icon:`fa-arrow-right` :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation
```